examples/demo/src/dashboard/Dashboard.tsx

```diff
@@ -9,6 +15,28 @@ import PendingOrders from './PendingOrders';
 import PendingReviews from './PendingReviews';
 import NewCustomers from './NewCustomers';
 
+import { Customer, Order, Review } from '../types';
+
+interface OrderStats {
+    revenue: number;
+    nbNewOrders: number;
+    pendingOrders: Order[];
+}
+
+interface CustomerData {
+    [key: string]: Customer;
+}
+
+interface State {
+    nbNewOrders: number;
```

Comment:
These must be optional, otherwise the initial state (`{}`) won't compile.

Reply:
Done!

Reply:
Interestingly I didn't get any warnings about them in development, I used `make run-demo` command to run it locally

Reply:
TypeScript is overrated :man_shrugging: